homeassistant/components/bthome/binary_sensor.py

```diff
@@ -0,0 +1,93 @@
+"""Support for BTHome binary sensors."""
+from __future__ import annotations
+
+from typing import Optional
+
+from bthome_ble import BTHOME_BINARY_SENSORS, SensorUpdate
+
+from homeassistant import config_entries
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.components.bluetooth.passive_update_processor import (
+    PassiveBluetoothDataProcessor,
+    PassiveBluetoothDataUpdate,
+    PassiveBluetoothProcessorCoordinator,
+    PassiveBluetoothProcessorEntity,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .device import device_key_to_bluetooth_entity_key, sensor_device_info_to_hass
+
+BINARY_SENSOR_DESCRIPTIONS = {}
+for key in BTHOME_BINARY_SENSORS:
```

Comment:
Please list the descriptions explicitly instead. We want to see what entities are created here in the integration code.